tests/cmds/test_departing_employee.py

```diff
@@ -337,3 +338,29 @@ def test_remove_departing_employee_when_user_not_on_list_prints_expected_error(
         )
         in result.output
     )
+
+
+@pytest.mark.parametrize(
+    "command, error_msg",
+    [
+        ("{} add".format(DEPARTING_EMPLOYEE_COMMAND), "Missing argument 'USERNAME'."),
+        (
+            "{} remove".format(DEPARTING_EMPLOYEE_COMMAND),
+            "Missing argument 'USERNAME'.",
```

Comment:
Opportunity to re-use error messages or have a helper method for creating them like

```
_get_err_opt_text("username")
_get_err_arg_text("CSV_FILE'")
```

`return "Missing option '--{}' .".format(opt)`

Reply:
This is not a mandatory suggestion!  Just something I noticed.